docs/README-migration.md

```diff
@@ -892,6 +892,16 @@ The following functions have been removed:
 * SDL_SensorGetDeviceType() - replaced with SDL_GetSensorInstanceType()
 * SDL_UnlockSensors()
 
+## SDL_shape.h

```

Comment:
I think the migration guide changes here are accidentally in this commit / branch instead of the remove shape api one.

Reply:
yes, moving to the other PR